composer/datasets/imagenet.py

```diff
@@ -146,7 +146,7 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataloaderHpara
 class Imagenet1kWebDatasetHparams(WebDatasetHparams):
     """Defines an instance of the ImageNet-1k dataset for image classification.
 
-    Parameters:
+    Args:
```

Comment:
References to `DataSpec` in the "Returns" section for the `initialize_object` method aren't resolving.